vllm/worker/model_runner.py

```diff
@@ -1037,9 +1037,17 @@ def load_model(self) -> None:
             assert supports_lora(
                 self.model
             ), f"{self.model.__class__.__name__} does not support LoRA yet."
+
             if supports_multimodal(self.model):
                 logger.warning("Regarding multimodal models, vLLM currently "
                                "only supports adding LoRA to language model.")
+            # It's necessary to distinguish between the max_position_embeddings
+            # of VLMs and LLMs.
+            if hasattr(self.model.config, "max_position_embeddings"):
```

Comment:
Hmm, why not use `supports_multimodal` to distinguish the two cases?

Reply:
Because MiniCPMV's config does not satisfy this condition
```python
MiniCPMVConfig {
  "_name_or_path": "MiniCPM/MiniCPM-V_2_6_awq_int4",
  "architectures": [
    "MiniCPMV"
  ],
  "attention_dropout": 0.0,
  "auto_map": {
    "AutoConfig": "configuration_minicpm.MiniCPMVConfig",
    "AutoModel": "modeling_minicpmv.MiniCPMV",
    "AutoModelForCausalLM": "modeling_minicpmv.MiniCPMV"
  },
  "batch_vision_input": true,
  "bos_token_id": 151643,
  "drop_vision_last_layer": false,
  "eos_token_id": 151645,
  "hidden_act": "silu",
  "hidden_size": 3584,
  "image_size": 448,
  "initializer_range": 0.02,
  "intermediate_size": 18944,
  "max_position_embeddings": 32768,
  "max_window_layers": 28,
  "model_type": "minicpmv",
  "num_attention_heads": 28,
  "num_hidden_layers": 28,
  "num_key_value_heads": 4,
  "patch_size": 14,
  "quantization_config": {
    "bits": 4,
    "group_size": 128,
    "modules_to_not_convert": null,
    "quant_method": "awq",
    "version": "gemm",
    "zero_point": true
  },
  "query_num": 64,
  "rms_norm_eps": 1e-06,
  "rope_scaling": null,
  "rope_theta": 1000000.0,
  "slice_config": {
    "max_slice_nums": 9,
    "model_type": "minicpmv"
  },
  "slice_mode": true,
  "sliding_window": null,
  "tie_word_embeddings": false,
  "torch_dtype": "float16",
  "transformers_version": "4.45.0",
  "use_cache": true,
  "use_image_id": true,
  "use_sliding_window": false,
  "version": 2.6,
  "vision_batch_size": 16,
  "vision_config": {
    "hidden_size": 1152,
    "image_size": 980,
    "intermediate_size": 4304,
    "model_type": "siglip_vision_model",
    "num_attention_heads": 16,
    "num_hidden_layers": 27,
    "patch_size": 14
  },
  "vocab_size": 151666
}

```